### ISSUE TITLE
Add Support for Visual Studio 2015 Shell

### ISSUE THREAD

**Issue by scott-xu** [Microsoft/PTVS#2156]
> PTVS 2.2.6 requires Visual Studio 2015 Community or higher, Express for Desktop or Express for Web. Update 3 is recommended.

Since Visual Studio 2015 Express Edition is not available anymore. And Visual Studio Community's license is not suit for my case. I'm wondering could PTVS add support for Visual Studio 2015 Shell?

- [Integrated Shell](http://go.microsoft.com/fwlink/?LinkId=615450)
- [Isolated Shell](http://go.microsoft.com/fwlink/?LinkId=615451)

**Comment by huguesv** [comment]
I'm sorry, our current plans with regards to VS 2015 support are to release critical bug fixes only.

We used to support the integrated shell in the past, but the testing matrix was getting too large and we decided to drop it when we added support for the Express editions.

I searched a bit, and it looks like Express is still available, they just made it harder to find.

Scroll down near the bottom of the page where it says "Still want Visual Studio Express?"
https://www.visualstudio.com/vs/visual-studio-express/
